.github/workflows/branch-protection.yml

```
@@ -0,0 +1,74 @@
+  push:
+    branches: [ main, develop ]
+  
+  # Também pode ser executado manualmente se necessário
+  workflow_dispatch:
+
+jobs:
+  protect-main-branch:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Configurar proteção da branch main
+        uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)